packages/sidenav/src/components/SideBar/Sidebar.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import SideBarStyled from "./SideBar.styled";
+
+export const SideBar = props => <SideBarStyled {...props} />;
```

Comment:
We don't need this file.
Exporting `SideBar.styled` as default on `index` would be enough.